smithy-swift-codegen/src/test/kotlin/IdempotencyTokenTraitTests.kt

```diff
@@ -6,34 +6,7 @@ class IdempotencyTokenTraitTests {
         val context = setupTests("Isolated/idempotencyToken.smithy", "aws.protocoltests.restxml#RestXml")
         val contents = getFileContents(context.manifest, "/RestXml/RestXmlProtocolClient.swift")
         val expectedContents = """
-    public func idempotencyTokenWithStructure(input: IdempotencyTokenWithStructureInput) async throws -> IdempotencyTokenWithStructureOutput {
```

Comment:
Like the test above, the expectation is way way broader than "generates idempotent middleware".